doc-content/enterprise-only/openshift/operator-deploy-basic-proc.adoc

```diff
@@ -17,8 +17,7 @@ ifdef::PAM[]
 +
 [IMPORTANT]
 ====
-In {PRODUCT} {ENTERPRISE_VERSION}, high-availability {CENTRAL} functionality deployment using the operator is for Technology Preview only. For more information about Red Hat Technology Preview features, see https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope]. For a fully supported high-availability deployment, use the high-availability authoring template on {OPENSHIFT} version 3.11. For instructions about deploying this template, see xref:assembly-openshift-templates[].
-====
+In {PRODUCT} {ENTERPRISE_VERSION}, high-availability {CENTRAL} functionality deployment using the operator is for Technology Preview only. For more information about Red Hat Technology Preview features, see https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope].
```

Comment:
shoudn't this url be pointing to the ibm website?

Reply:
I removed the link.